taskcluster/ci/signing/kind.yml

```diff
@@ -24,21 +24,24 @@ only-for-build-types:
     - android-armv7/release
     - linux/opt
     - macos/opt
+    - windows/opt
 
 job-template:
-    description: Sign MozillaVPN
-    worker: {} # Will be configured by signing:transforms
+    description: sign mozillavpn
     run-on-tasks-for: []
     add-index-routes: build
     signing-format:
         by-build-type:
             android.*: autograph_apk
             linux.*: autograph_debsign
             macos.*: macapp
-            default: ""
+            windows/opt: autograph_authenticode
```

Comment:
Should we still have a default?

Reply:
Nah I don't think so. If we ever add another platform to `only-for-build-types`, it would be invalid to have an empty signing-format anyway, so throwing an error (due to not being matched here) seems like a better outcome anyway.